docs/source/index.rst

```diff
@@ -1,9 +1,85 @@
 .. Copyright (c) 2022 - 2023, Oracle and/or its affiliates. All rights reserved.
 .. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
+.. _index:
+
+.. meta::
+   :description: macaron - A CI/CD security analysis tool for supply-chain attacks
+   :keywords: CI/CD, SLSA, supply-chain security
+
 Macaron documentation
 =====================
 
+Software supply-chain attacks are becoming more prevalent as the systems get more complex, particularly with respect to the use of open-source
+third-party code. Attacks include stealing credentials, tampering with the
+code, tampering with the code repository, and tampering with the build system.
+It is crucial to have guarantees that the third-party code we rely on is the
+code we expect.
+
+To tackle these problems, `Supply-chain Levels for Software Artifacts (SLSA or "salsa") <https://slsa.dev>`_ is created to improve the integrity and
+protection of the software supply-chain. Macaron can analyze a software
+repository to determine its SLSA level and provide supply-chain transparency of the build process.
+
+Overview
+********
```

Comment:
We are using a different style for other RST (`=` for first header and `-` for second header).

Reply:
See commit [e09d667](https://github.com/oracle-samples/macaron/pull/284/commits/e09d6673e5a8ef9696364847d86137435ffe8a2b)